internal/namespaces/instance/v1/command_server_create_test.go

```diff
@@ -0,0 +1,216 @@
+package instance
+
+import (
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+)
+
+func Test_CreateServer(t *testing.T) {
+
+	//t.Run("Simple", core.Test(&core.TestConfig{
+	//	Commands: GetCommands(),
+	//	BeforeFunc: func(ctx *core.BeforeFuncCtx) error {
+	//		ctx.Meta["Server"] = ctx.ExecuteCmd("scw instance server create image=ubuntu_bionic")
+	//		return nil
+	//	},
+	//	Cmd: "scw instance server create image=ubuntu_bionic",
+	//	AfterFunc: func(ctx *core.AfterFuncCtx) error {
+	//		ctx.ExecuteCmd("scw instance server delete server-id={{ .Server.id }}")
+	//		return nil
+	//	},
+	//	Check: core.TestCheckCombine(
+	//		core.TestCheckGolden(),
+	//	),
+	//}))
+}
+
+func Test_CreateServerErrors(t *testing.T) {
```

Comment:
If I understand well, you never remove created instances?

Reply:
None of this test succeed to create an instance. (the test fail if it does)